README.md

```diff
@@ -121,6 +121,86 @@ Build for the web with [webpack](https://webpack.js.org/) ([example](./example/w
 
 **Important:** make sure to copy `voyager.worker.js` from `node_modules/graphql-voyager/dist` to the same folder as your main bundle.
 
+## Middleware
+Graphql Voyager has middleware for next frameworks:
+
+### Express
+
+#### Properties
+Middleware for Express supports following properties:
```

Comment:
Minor grammar, change to: `Express middleware supports the following properties:`